component/component.go

```diff
@@ -29,6 +29,11 @@ type Component interface {
 	// Start() then the collector startup will be aborted.
 	// If this is an exporter component it may prepare for exporting
 	// by connecting to the endpoint.
+	//
+	// The context passed to this method must be used only in the scope of Start, and
+	// should not be stored or reused in any async work that the component executes
+	// after the start. It is important because the context may be cancelled by the
+	// service at any moment between Start and Shutdown.
```

Comment:
I think if we add the above correction we no longer need this.

Reply:
done.